docs/resources/packer_channel.md

```diff
@@ -11,45 +11,42 @@ The Packer Channel resource allows you to manage image bucket channels within an
 
 ## Example Usage
 
-To create a channel with no assigned iteration.
+To create a channel.
 ```terraform
 resource "hcp_packer_channel" "staging" {
   name        = "staging"
   bucket_name = "alpine"
 }
 ```
 
-To create, or update an existing, channel with an assigned iteration.
+To create a channel with a Terraform-managed iteration.
```

Comment:
The first time a read I thought the iteration was managed by terraform (which doesn't make sense) but it is actually the action of assigning an iteration that is. What do you think of making it more specific?  
```suggestion
To create a channel with iteration assignment managed by Terraform.
```

Reply:
Agreed, implemented this suggestion